library/src/scala/quoted/reflect.scala

```diff
@@ -0,0 +1,18 @@
+package scala.quoted
+
+/** Current QuoteContext.reflect in scope
+ *
+ *  Usage:
+ *  To use reflection inside a method
+ *  ```
+ *  def f(using QuoteContext) = { // or (using qctx: QuoteContext)
+ *    import reflect._ // equivalent to import reflect._
+ *    ...
+ *  }
+ *  ```
+ *  or to use reflection in the signature of a method
+ *  ```
+ *  def f(using QuoteContext)(tree: reflect.Tree): reflect.Position = ...
+ *  ```
+ */
+def reflect(using qctx: QuoteContext): qctx.reflect.type = qctx.reflect
```

Comment:
What about #10295 ? There is no precedent in the language where we can _import_ an unstable prefix. Or at least, with inline and checks?

Meanwhile, it seems to me that `import qctx.reflect._` is better than `import reflect._`, as the former tells some information about the relationship between `qctx` and `reflect`, while the latter makes me wonder what's the magic behind.